sdks/python/setup.py

```diff
@@ -300,25 +296,25 @@ def get_portability_package_data():
             # https://github.com/googleapis/google-cloud-python/issues/10566
             'google-auth>=1.18.0,<3',
             'google-auth-httplib2>=0.1.0,<0.2.0',
-            'google-cloud-datastore>=1.8.0,<2',
+            'google-cloud-datastore>=1.8.0,<3',
             'google-cloud-pubsub>=2.1.0,<3',
             'google-cloud-pubsublite>=1.2.0,<2',
             # GCP packages required by tests
             'google-cloud-bigquery>=1.6.0,<4',
-            'google-cloud-bigquery-storage>=2.6.3,<2.17',
+            'google-cloud-bigquery-storage>=2.6.3,<3',
             'google-cloud-core>=0.28.1,<3',
-            'google-cloud-bigtable>=0.31.1,<2',
+            'google-cloud-bigtable>=0.31.1,<3',
```

Comment:
let's increase `google-cloud-bigtable>=2` for similar reasons, v2 has been longer than 1 year.

Reply:
cc: @johnjcasey FYI.

Reply:
Okay. I haven't gone through all GCP deps yet because they didn't fail the code when we update dependencies. But it would be good to have latest bounds for GCP dependencies since many of them have had some breaking changes in the past.